hipogram/posts/views.py

```diff
@@ -18,14 +19,21 @@ class PostListView(ListView):
     template_name = "post_list.html"
     paginate_by = 2
 
+def post_filter_list_view(request):
+    breakpoint()
+    post = get_object_or_404(Post)
+
+    if request.method == 'POST':
+        ...
+
```

Comment:
We decided to wait for this one, and also it doesn't related to current PR, please remove this.

Reply:
One more mistake, this part has a breakpoint which shouldn't be in anywhere except your local environment.